WePool/src/app/app.component.ts

```diff
@@ -6,5 +6,7 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'WePool';
+
+  questions = [{question: "When I ride, I am talkative and love to catch up with my coworkers about their out of work happenings! When I ride, I am talkative and love to catch up with my coworkers about their out of work happenings! When I ride, I am talkative and love to catch up with my coworkers about their out of work happenings!", type: "checkbox"}, 
```

Comment:
Is this question text supposed to be repeated like that?

Reply:
Yes, it was just an example to see the text wrapping.